hw/top_earlgrey/data/xbar_main.hjson

```diff
@@ -158,19 +157,19 @@
     },
   ],
   connections: {
-    corei:  ["rom_ctrl.rom", "rv_dm.rom", "ram_main", "eflash"],
-    cored:  [
+    rv_core_ibex.corei:  ["rom_ctrl.rom", "rv_dm.rom", "ram_main", "eflash"],
+    rv_core_ibex.cored:  [
       "rom_ctrl.rom", "rom_ctrl.regs", "rv_dm.rom", "rv_dm.regs",
       "ram_main", "eflash", "peri", "flash_ctrl.core", "flash_ctrl.prim",
       "aes", "entropy_src", "csrng", "edn0", "edn1", "hmac",
       "rv_plic", "otbn", "keymgr", "kmac", "sram_ctrl_main",
-      "rv_core_ibex_peri"
+      "rv_core_ibex.reg"
```

Comment:
might change this to `rv_core_ibex.cfg`...because `u_reg_reg` just sounds awkward...